defs/scenarios/storage/ceph/ceph-mon/mon_db_too_big.yaml

```diff
@@ -0,0 +1,21 @@
+checks:
+  mon_large_db:
+    input:
+      command: ceph_health_detail_json_decoded
+    expr: '.*mon .+ is using a lot of disk space.*'
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[eq, HEALTH_WARN]]
```

Comment:
do you think its worth doing ```[[ne, HEALTH_OK]]``` since maybe its possible for the cluster to be HEALTH_ERR due to another issue and also exhibiting this problem?

Reply:
ack, making this change